tests/pyrog/api/test_column.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from faker import Faker
+
+from django.urls import reverse
+
+faker = Faker()
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.parametrize("table, column_field, status_code", [(faker.word(), faker.word(), 201)])
+def test_create_column(
+    api_client,
+    join,
+    input_factory,
+    owner,
+    table,
+    column_field,
+    status_code,
+):
+    url = reverse("columns-list")
+
+    data = {
+        "join": join.id,
+        "input": input_factory().id,
+        "owner": owner.id,
+        "table": table,
+        "column": column_field,
+    }
+    response = api_client.post(url, data)
+
+    assert response.status_code == status_code
+
+
+@pytest.fixture(params=[True, False], ids=["With join", "Without join"])
+def x_column(request, column_factory):
```

Comment:
why is this fixture only used in test_retrieve ? sounds like it would be a good idea to test both "kinds" of colums for every operation

Reply:
We may... But it might be hard to maintain depending on the logic and it will currently add no real value to the test (we have not yet specify a different (api) behaviour with/without join). I think it's best to leave the some things *unspecified* (hence no tests), until we hit a specific need.

Reply:
gotcha (and agreed)